### PR TITLE
bump introduction one level up

### PR DESCRIPTION
**Pull request by nonsense**

### CODE REVIEW

File: contents/installation.rst [hunk 1]
```diff
@@ -10,18 +10,16 @@ Swarm runs on all major platforms (Linux, macOS, Windows, Raspberry Pi, Android,
 ..  note::
   The swarm package has not been extensively tested on platforms other than Linux and macOS.
 
-Installing Swarm from a package manager
-=======================================
+Installing Swarm on Ubuntu via PPA
+==================================
 
-Install on Ubuntu via PPAs
```
Comment (nonsense) [inline contents/installation.rst]: No need for so many nested levels within the Installation section.

File: contents/introduction.rst [hunk 2]
```diff
@@ -80,29 +80,29 @@ Code
...
 Roadmap
----------------
+=======
 
 Roadmap is located at https://github.com/ethersphere/Swarm/wiki/roadmap
 
 .. note:: Swarm is experimental code and untested in the wild. Use with extreme care. We encourage developers to connect to the testnet with their permanent nodes and give us feedback.
 
 Public gateways
--------------------------------
```
Comment (nonsense) [inline contents/introduction.rst]: Roadmap, Public gateways, Example Dapps, and reporting/contributing, should not be under the `Code` section, but alongside it.

File: contents/introduction.rst [hunk 3]
```diff
@@ -43,7 +43,7 @@ The Ethereum Foundation operates a Swarm testnet that can be used to test out fu
 Everyone can join the network by running the Swarm client node on their server, desktop, laptop or mobile device. See :ref:`Getting Started` for how to do this.
 The Swarm client is part of the Ethereum stack, the reference implementation is written in golang and found under the go-ethereum repository. Currently at POC (proof of concept) version 0.3 is running on all nodes.
 
-Swarm offers a **local HTTP proxy** API that dapps or command line tools can use to interact with Swarm. Some modules like `messaging  <PSS>`_ are   only available through RPC-JSON API. The foundation servers on the testnet are offering public gateways, which serve to easily demonstrate functionality and allow free access so that people can try Swarm without even running their own node.
+Swarm offers a **local HTTP interface** API that dapps or command line tools can use to interact with Swarm. Some modules like `messaging  <PSS>`_ are   only available through RPC-JSON API. The foundation servers on the testnet are offering public gateways, which serve to easily demonstrate functionality and allow free access so that people can try Swarm without even running their own node.
```
Comment (nonsense) [inline contents/introduction.rst]: I think interface is better than proxy, as proxy generally means only forwarding, and this is not what our HTTP API is doing.

Comment (justelad) [reply contents/introduction.rst]: well.... we've been referring to it so far as a proxy... i think it shows up as one on several locations in the docs too...

Comment (nonsense) [reply contents/introduction.rst]: On a second thought, maybe it is best to leave it as `proxy`. I will revert this change.

**Pull request merged**